lib/Imgur/Api/Image.php

```diff
@@ -49,7 +49,7 @@ public function upload($data)
         }
 
         if ('file' === $data['type']) {
-            $data['image'] = '@' . $data['image'];
```

Comment:
The `@` tricks is working with curl https://www.iditect.com/how-to/56655862.html
But I don't know with Guzzle.